modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/IgnitePdsCacheStartStopWithFreqCheckpointTest.java

```diff
@@ -84,7 +83,7 @@ private CacheConfiguration cacheConfiguration(int cacheIdx) {
             .setAtomicityMode(CacheAtomicityMode.TRANSACTIONAL)
             .setCacheMode(CacheMode.REPLICATED)
             .setBackups(0)
-            .setRebalanceMode(CacheRebalanceMode.NONE);
+            .setRebalanceDelay(Long.MAX_VALUE);
```

Comment:
RebalanceMode.None deprecated in Mvcc mode only.
So, change in tests should not affect non-mvcc scenarios.

If is possible to set proper mode if MvccFeatureChecker.forceMvcc() == true?
Or rewrite rebalance mode in MvccProcessor like we do for Transactional atomicity mode?